src/data/faq_de.json

```diff
@@ -146,6 +146,23 @@
                             "Unter dem Reiter Zertifikate kann die App nur '<b>EU Digitale</b> COVID-Zertifikate' scannen.",
                             "Siehe auch folgenden Eintrag zu <a href='#vac_cert_invalid' target='_self' rel='noopener'>ungültigen Impfzertifikaten</a>."
                          ]
+                    },
+                    {
+                        "title": "Wie lange ist mein Zertifikat gültig?​",
+                        "anchor": "eu_dcc_validity",
+                        "active": true,
+                        "textblock": [
+                            "Der folgende Text bezieht sich auf in Deutschland ausgestellte digitale COVID Zertifikate der EU, d.&nbsp;h. Impf-, Test-, und Genesenenzertifikate. Die angegeben Zahlen sind der aktuelle Stand von Juli 2021.",
+                            "Wir unterscheiden grundsätzlich​<ul><li>die Gültigkeit des digitalen Zertifikats (die <b>technische Gültigkeit</b>) und</li><li>die Gültigkeit des Impfschutzes (die <b>fachliche Gültigkeit</b>) im Sinne der 'Dauer des Schutzes'</li></ul>",
+                            "<b>Technische Gültigkeit</b>",
+                            "Das Zertifikat als Dokument hat ein Ausstellungsdatum und ein technisches Ablaufdatum, so wie auch Ihr Reisepass oder Personalausweis beispielsweise. Für alle Zertifikate gilt, dass das Ablaufdatum vom Herausgeber des Zertifikats gesetzt wird. Derzeit werden in Deutschland die digitalen Impf-, Test- und Genesenen-Zertifikate der EU in der Regel mit einer technischen Gültigkeit von einem Jahr ab Ausstellungsdatum ausgestellt.​",
+                            "<b>Fachliche Gültigkeit</b>",
+                            "In dem Zertifikat werden Daten transportiert, z.&nbsp;B. in dem Fall des Impfzertifikats, Informationen zur Impfung. Diese Impfung fand an einem Datum X statt und gilt ab dann, nach aktuellem Stand der wissenschaftlichen Erkenntnisse, z.&nbsp;B. in Deutschland ein Jahr. Danach gilt der Impfschutz als erloschen. Die Dauer des Impfschutzes bezeichnen wir als 'fachliche Gültigkeit'. Die fachliche Gültigkeit der Impfung zählt also ab dem letzten Impfdatum bzw. genau genommen, ab dem Tag nach Erreichen der Wirksamkeit des Impfschutzes (14 Tage nach Datum X).​ Während in Deutschland derzeit die fachliche Gültigkeit auf ein Jahr festgelegt ist, kann die Gültigkeit des Impfschutzes in anderen Ländern davon abweichen.​",
+                            "Bei einem​ Genesenen-Zertifikat ist die fachliche Gültigkeit laut des EU eHealth Networks festgelegt auf 'Datum des ersten positiven PCR-Test-Ergebnisses der Person' plus 180 Tage.​",

```

Comment:
The screen shot in the announcement for 2.5 
https://www.coronawarn.app/de/blog/2021-07-12-cwa-version-2-5/
shows that the app shows the term "Genesenenzertifikat".
not "Genesenen-Zertifikat".

Reply:
ok, changed all occurrences to Genesenenzertifikat